script.js

```diff
@@ -1,406 +1,205 @@
-      '1 teaspoon fennel seed, toasted',
-      '1 teaspoon cumin pods, toasted',
-      '1 teaspoon red pepper flakes',
-      '1/2 teaspoon dried oregano',
-      '2 medium cloves garlic, minced (about 2 teaspoons)',
-      '2 tablespoons vegetable or canola oil',
-      '1 whole flap meat steak, 2 to 2 1/2 pounds',
-      'Kosher salt'
-    ],
-    source: 'Serious Eats',
-    totalTime: 240,
-    url: 'http://www.seriouseats.com/recipes/2012/05/spice-rubbed-grilled-flap-meat-sirloin-tip-recipe.html',
-    image: './recipe-images/grilled.jpg'
-  }
-]
+//DOM selectors
+const cards = document.querySelector(".show-cards");
+const filterAll = document.getElementById("all");
+const filterDrama = document.getElementById("drama");
+const filterComedy = document.getElementById("comedy");
+const filterCrime = document.getElementById("crime");
+const sortAZ = document.getElementById("az");
+const sortRelease = document.getElementById("release");
+const sortRating = document.getElementById("rating");
+const randomButton = document.getElementById("random");
+
+//Load array content into HTML, this will fill the show cards with their respective information
+const loadShows = (shows) => {
+  cards.innerHTML = "";
+  shows.forEach((show) => {
+    cards.innerHTML += `
+    <div class="show-cards">
+      <img class="card-img" src="${show.image}">
+      <h2>${show.title}</h2>
+      <p class="card-info">${show.creator}</p>
+      <p class="card-info">${show.year}</p>
+      <p class="card-info">${show.genre}</p>
+      <p class="card-info">${show.rating}</p>
+      <p class="description">${show.description}</p>
+    </div>
+  `;
+  });
+};
+
+//Load cards when page opens
+loadShows(shows);
+
+//Declare lowToHigh variable for sort functin to work
+let lowToHigh = true;
+
+//Make buttons return to default styling after clicking
+document.querySelectorAll('.buttons').forEach(button => {
+  button.addEventListener('click', () => {
+    button.blur();
+  });
+});
+
+//Function to filter by genre
+const filterByGenre = (shows, genre) => {
+  return shows.filter(show => show.genre.toLowerCase() === genre.toLowerCase());
+};
+
+//Functions to sort by rating, year or alphabeticaly
+const sortByRating = (shows, ascending = true) => {
+  return shows.sort((a, b) => {
+    return ascending ? a.rating - b.rating : b.rating - a.rating;
+  });
+};
+
+const sortByYear = (shows, ascending = true) => {
+  return shows.sort((a, b) => {
+    return ascending ? a.year - b.year : b.year - a.year;
+  });
+};
+
+const sortByAZ = (shows, ascending = true) => {
+  return shows.sort((a, b) => {
+    const titleA = a.title.toLowerCase();
+    const titleB = b.title.toLowerCase();
+    return ascending ? titleA.localeCompare(titleB) : titleB.localeCompare(titleA);
+  });
+};
+
+//Function and EventListener for randomizer
+const getRandomShow = () => {
+  const randomIndex = Math.floor(Math.random() * shows.length);
+  return shows[randomIndex];
+};
+
```

Comment:
String interpolation is to the point. The code is clean and easy to read. Well done!